internal/v1/api.yaml

```diff
@@ -2092,6 +2092,11 @@ components:
           example: true
           description: |
             Optional flag to use rhc to register the system, which also always enables Insights.
+        insights_client_proxy:
+          type: string
+          format: uri
+          description: |
+            Optional value to set proxy option when registering the system to Insights.
```

Comment:
Not sure if it makes sense, but I added this to get the unit tests to pass

Reply:
The description? Probably just forgot to run `tools/prepare-source`?